Source/JavaScriptCore/jit/ThunkGenerators.cpp

```diff
@@ -626,6 +626,11 @@ MacroAssemblerCodeRef<JITThunkPtrTag> stringGetByValGenerator(VM& vm)
     return FINALIZE_THUNK(patchBuffer, JITThunkPtrTag, "string_get_by_val"_s, "String get_by_val stub");
 }
 
+enum class RelativeNegativeIndex : bool {
+    No,
+    Yes,
+};
```

Comment:
Usually No, Yes enums are a single line.

Reply:
Thank you, Fixed!